### PR TITLE
Add fabtotum materials

### PR DESCRIPTION
**Pull request by krios-fabteam**

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Devs, see CURA-4128.
This Pull Request should be merged at the same time as this one: https://github.com/Ultimaker/Cura/pull/2162

File: fabtotum_abs_black.xml.fdm_material [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+  <metadata>
+    <name>
+      <brand>FABtotum</brand>
+      <material>ABS</material>
+      <color>Carbon Black</color>
+    </name>
+    <color_code>#000000</color_code>
+	<description>FABtotum ABS black profile.</description>
```
Comment (Ghostkeeper) [inline fabtotum_abs_black.xml.fdm_material]: Please always indent with 4 spaces per step. You're mixing tabs and spaces here, too.

**Comment by PaoloP74** [comment]
Need add version numbers to XML material file in according to the new v1.3 of the fdm_material spec.
from: <fdmmaterial xmlns="http://www.ultimaker.com/material">
to: <fdmmaterial xmlns="http://www.ultimaker.com/material" version="1.3">

**Comment by alekseisasin** [comment]
PaoloP74, Please add GUID (unique generated) and version element for all your material types.
<GUID></GUID>
<version></version>

**Comment by PaoloP74** [comment]
@alekseisasin I'm not a FabTeam Developer but an independent maker. Ask @krios-fabteam.

**Comment by krios-fabteam** [comment]
@alekseisasin  there are some specifics for the GUID generation?

**Comment by fieldOfView** [comment]
You can create GUIDs here for example:
https://guidgenerator.com/
The idea is that the chance of a "collision" between GUIDs created anywhere using the type of algorithms used by that website, or other implementations of GUID generators, is negligible.

**Comment by alekseisasin** [comment]
PaoloP74, I checked all your profiles again. I noticed that you added couple of ABS and PLA material types which settings are same and they differ only by color. 

I would like to ask you to create generic material profile for ABS and PLA, like "generic_pla_fabtotum".

**Comment by PaoloP74** [comment]
@krios-fabteam, see alekseisin comment.

**Comment by krios-fabteam** [comment]
Hi @alekseisasin,
if i create a generic profile (es. `generic_pla_fabtotum.xml.fdm_material`) how i can then set the colors?
I've noticed that for example Ultimaker use a specific file for color

**Comment by nallath** [comment]
A generic material doesn't really have a color, as it's a "generic" representation of that material. Ultimaker did add files for every color, as our spools have NFC tags, which are matched by the GUID.

**Comment by krios-fabteam** [comment]
So I have to remove all the profiles and leave only the generic version for PLA and ABS without specifying the color?

**Comment by alekseisasin** [comment]
Yes. If you don’t have any specific settings then please remove them.
You will have next files:
1 → generic PLA
2 → generic ABS
3 → generic nylon

**Comment by krios-fabteam** [comment]
hope is ok with the last commit

**Comment by alekseisasin** [comment]
It is almost done, the file names do not match to the definition in quality profiles. material = generic_fabtotum_abs
I will do these changes for you.

**Comment by krios-fabteam** [comment]
Thanks guys!

**Pull request merged**